### PR TITLE
[Docbug][Composemanifestcode][4185]

### PR DESCRIPTION
**Pull request by v-vidhyarani**
replaced the code in the right place.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fec77b144727030f41cf64fe00cbf6d291a91396/msteams-platform/resources/schema/manifest-schema.md) | 96 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2d56f3e5-405e-45dc-854a-71b744267785) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-abirade — APPROVED** [review]
Approved.

**Review by v-abirade — DISMISSED** [review]
Approved.

File: msteams-platform/resources/schema/manifest-schema.md [hunk 1]
```diff
@@ -178,6 +178,12 @@ The following schema sample shows all extensibility options:
                   "value": "Value of the choice"
                 }
               ]
+               "taskInfo": {
```
Comment (gnanasuriyan-gnanaoli) [inline msteams-platform/resources/schema/manifest-schema.md:181]: Hi, this is still the wrong place and comma is missing(so the json will fail the validation).

Please check this comment to add `taskInfo` to correct place
https://github.com/MicrosoftDocs/msteams-docs/issues/4185#issuecomment-922843921

Comment (Wajeed-msft) [reply msteams-platform/resources/schema/manifest-schema.md:181]: @Meghana-MSFT - could you please check the payload once?

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ebbeaf2998bba1a6d45ec9a06445fc51b0f72ecb/msteams-platform/resources/schema/manifest-schema.md) | 96 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6a439db0-b48a-4bec-a431-0381c3e25d8b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**